tests/performance/apdex-score/test/pageobjects/reports.page.js

```diff
@@ -0,0 +1,34 @@
+const { $ } = require('@wdio/globals');
+const Page = require('./page');
+
+class ReportsPage extends Page {
+    
+  get iconHealthCare () {
+    return $('(//*[@text="icon-healthcare"])[1]');
+  }
+
+  get itemFirstReport () {
+    return $('((//android.widget.ListView//android.view.View)[1]//android.view.View)[1]');
+  }
+
+  get iconBack () {
+    return $('//*[@text="Back"]');
+  }
+
+  async viewAReport () {
```

Comment:
I'm not sure what the difference between `viewAReport` and `viewAReportNE` is. Why can't we use the same function for all?